### PR TITLE
Added a new file with some utils I use to be able to do a switch with strings

### PR DESCRIPTION
**Pull request by maitesin**
I think I can still improve some of the content of the cases (I can see three really similar).

If you like the way I make the switch I will spend some more time in other files to change big blocks of if else if... that can be converted into switch.

### CODE REVIEW

File: src/Utils.hpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ *  This file is part of the mhwd - Manjaro Hardware Detection project
+ *
+ *  mhwd - Manjaro Hardware Detection
+ *  Roland Singer <roland@manjaro.org>
+ *  Łukasz Matysiak <december0123@gmail.com>
+ *  Filipe Marques <eagle.software3@gmail.com>
```
Comment (philmmanjaro) [inline src/Utils.hpp]: Please also add your author name to the list since it is your code.

**Comment by philmmanjaro** [comment]
Yes, seems fine go for it.

**Comment by maitesin** [comment]
I just added my name. Thanks for adding me to the Manjaro Linux organization. I will do my best!

**Comment by philmmanjaro** [comment]
Oscar have fun and welcome to the team.

**Pull request merged**